vendor/kmodules.xyz/client-go/core/v1/kubernetes.go

```diff
@@ -79,7 +79,7 @@ func UpsertContainer(containers []core.Container, upsert core.Container) []core.
 			return containers
 		}
 	}
-	return append(containers, upsert)
+	return append([]core.Container{upsert}, containers...)
```

Comment:
Please don't update vendor file. It will be overwritten when we run `go mod tidy`. Instead, change the order in calling function.